client/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022 Victoria Metrics Inc.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>io.victoriametrics.client</groupId>
+        <version>1.0-SNAPSHOT</version>
+        <artifactId>parent</artifactId>
+    </parent>
+
+    <artifactId>client-jvm</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
I remember we discussed that this lib is not really a "client", as it doesn't make any requests to server. It falls into "exposition" role, and probably can be depended on by other libs like real client libs that request queries, that we may add later. It probably can be named as "metrics" or "exposition" or "common" or something like that. I probably prefer "metrics".